benchmarks/test_peer_selection.py

```diff
@@ -22,81 +22,78 @@
     absolute_import, unicode_literals, print_function, division
 )
 
-import mock
-import random
+from tornado import gen, ioloop
```

Comment:
Yes, I understand that but this is measuring something different.

I want to leave the benchmark that measures the cost of calling `choose()` separate from your new benchmark. The cost of choose used to be `O(n)`, and now it is `O(1)`. Maybe in the future we change it to `O(log n)` because it allows us to optimize something else elsewhere in the system. I want this benchmark, that measures the cost of `choose()` to stay in the repo so that we can use it if necessary.

The new benchmark that makes end to end requests is a separate benchmark that goes in a different file.